bool follow_local_path
float32[] global_path_x
float32[] global_path_y
geometry_msgs/Point next_point